src/data/units.json

```diff
@@ -700,15 +700,19 @@
 			},
 			{
 				"title": "Fiery Touch",
-				"desc": "Scratches and burns a distant enemy. This attack has 2 hexagons extra reach.",
-				"info": "10 slash + 10 burn damage for each hit.",
-				"upgrade": "Double range for the burn hit.",
+				"desc": "Scratches and burns a distant enemy up to 3 hexagons away.",
```

Comment:
Usually struggling to formulate lines like this in order to fit properly on the card in 1 or 2 rows in English.
Will probably need to review and tweak (the ability tooltips kinda break those rows for a long time now).

Reply:
![CleanShot 2022-01-01 at 12 26 30](https://user-images.githubusercontent.com/199204/147842783-75805885-ea8f-48ab-a022-08fc4c15cf94.png)

Reply:
That one is good, though you modified description itself which is just 1 row and a half now xD